packages/tlschema/src/bindings/TLArrowBinding.ts

```diff
@@ -1,10 +1,18 @@
 import { T } from '@tldraw/validate'
 import { VecModel, vecModelValidator } from '../misc/geometry-types'
-import { createBindingPropsMigrationSequence } from '../records/TLBinding'
+import {
+	createBindingPropsMigrationIds,
+	createBindingPropsMigrationSequence,
+} from '../records/TLBinding'
 import { RecordProps } from '../recordsWithProps'
 import { arrowShapeVersions } from '../shapes/TLArrowShape'
 import { TLBaseBinding } from './TLBaseBinding'
 
+/** @public */
+export const ElbowArrowSnap = T.literalEnum('center', 'edge-point', 'edge', 'none')
```

Comment:
before i forget, one of these was maybe like a 'cardinal-point' or something? i forget